packages/salesforcedx-vscode-replay-debugger/src/states/frameEntryState.ts

```diff
@@ -8,21 +8,13 @@
 import { basename } from 'path';
 import { Source, StackFrame } from 'vscode-debugadapter';
 import Uri from 'vscode-uri';
-import { SFDC_TRIGGER } from '../constants';
 import { LogContext } from '../core/logContext';
 import { DebugLogState } from './debugLogState';
+import { FrameState } from './frameState';
 
-export class FrameEntryState implements DebugLogState {
-  private readonly signature: string;
-  private readonly frameName: string;
-
+export class FrameEntryState extends FrameState implements DebugLogState {
   constructor(fields: string[]) {
```

Comment:
Do you need this if it just calls `super`?

Reply:
I thought about that but I'd rather just keep it there in case we ever need to do something specific in either of the extending class constructors.